db/config.cc

```diff
@@ -909,6 +912,11 @@ db::config::config(std::shared_ptr<db::extensions> exts)
     , task_ttl_seconds(this, "task_ttl_in_seconds", liveness::LiveUpdate, value_status::Used, 10, "Time for which information about finished task stays in memory.")
     , cache_index_pages(this, "cache_index_pages", liveness::LiveUpdate, value_status::Used, false,
         "Keep SSTable index pages in the global cache after a SSTable read. Expected to improve performance for workloads with big partitions, but may degrade performance for workloads with small partitions.")
+    , auth_superuser_name(this, "auth_superuser_name", value_status::Used, "", 
+        "Initial authentication super username. Ignored if authentication tables already contain a super user")
+    , auth_superuser_salted_password(this, "auth_superuser_salted_password", value_status::Used, "", 
+        "Initial authentication super user salted password. Create using mkpassword or similar. The hashing algorithm used must be available on the node host. "
+        "Ignored if authentication tables already contain a super user password.")
```

Comment:
I'm not a fan of use-once options but don't have anything better to suggest (that preserves ease of use).

Reply:
Driven by 
1.) The need to specify an initial user for cert auth
2.) Complaints about the cassandra:cassandra default previously, i.e. time during which an authenticating node runs with a known user/passwd. 
Same reservations about use-once here, but I have no better alternative (considered adding yet another config file, but that does not really change anything).

Reply:
Oh I agree, just felt compelled to complain.